app/javascript/components/SweetPhoneInput/SweetPhoneInput.js

```diff
@@ -75,7 +75,7 @@ class SweetPhoneInput extends Component {
   onPhoneNumberChange = (value: string = '') => {
     this.setState(prevState => ({
       ...prevState,
-      phoneNumber: format(value, this.getCountryCode(), 'National'),
+      phoneNumber: new asYouType(this.getCountryCode()).input(value),
```

Comment:
beware of `asYouType` since it will insert brackets when the number is not yet complete, and it might re-introduce the bug where you can't delete past `+1 (413)` because deleting the bracket re-inserts it?